tasks/main.yml

```diff
@@ -130,6 +130,16 @@
 - name: clear out leftover build cruft from the project requirements
   file: path={{ venv_dir }}/build state=absent
 
+- name: "(ssh-agent hack: grant access to {{ project_user }})"
```

Comment:
This task occurs after the task to update the repository (which makes use of github_deploy_key). My initial thinking was I wouldn't need to set github_deploy_key if I enabled ssh agent forwarding, but currently both are required if the pip requirements file also has private git repositories. If ssh agent forwarding is enabled for the entire deploy session (so nodejs could also take advantage of it), perhaps it should be moved into common? Otherwise it would be helpful to document the interaction between github_deploy_key and ssh agent forwarding.

Reply:
Scratch that, common doesn't have project_user, so that wouldn't work.

Reply:
@copelco It turns out that tequila-common does create the project user, but it does so by always assuming that it is named `{{ project_name }}`.  That seems fragile and should change, since we have it independently configurable here in tequila-django.